### DISCUSSION TITLE
`hidden="until-found"` for DisclosureGroup

### DISCUSSION THREAD

**Discussion by sadeghbarati** [adobe/react-spectrum#7162] [Q&A] [answered]
Hi, I want to know if there is an option to enable this attribute for `Disclosure`s inside `DisclosureGroup`

https://github.com/adobe/react-spectrum/blob/533a02fe189de1406ced8306d213088fff0bee09/packages/%40react-aria/disclosure/src/useDisclosure.ts#L57

**Comment by reidbarber** [comment, marked as answer]
We're working to support this in `DisclosureGroup` if possible before GA of this component. It doesn't currently work because we are controlling expansion state at the group level for Disclosures used within a DisclosureGroup.
